README.md

```diff
@@ -139,6 +141,43 @@ attach to currently active session.
 *The R terminal used in the screenshot is [radian](https://github.com/randy3k/radian) which is cross-platform and
 supports syntax highlighting, auto-completion and many other features.*
 
+## Creating keybindings for R commands
+
+There are 3 ways you can use extension functions to create keybindings that run R commands in the terminal:
+
+1. `r.runCommand` to make a keybinding to run any R expression. 
+2. `r.runCommandWithEditorPath` to create a keybinding for an R expression where the placeholder value `$$` is interpolated with the current file path.
+3. `runCommandWithSelectionOrWord` to create a keybinding for an R expression where `$$` is interpolated with the current selection or the current word the cursor is on.
+
+Here are some example entries from `keybindings.json`:
+
+```
+[
+    {
+        "description": "run drake::r_make()",
+        "key": "ctrl+;",
+        "command": "r.runCommand",
+        "when": "editorTextFocus",
+        "args": "drake::r_make()"
+    },
+    {
+        "description": "load drake target at cursor",
+        "key": "ctrl+shift+;",
+        "command": "r.runCommandWithSelectionOrWord",
+        "when": "editorTextFocus",
+        "args": "drake::loadd($$)"
+    },
+    {
+        "description": "knit to html",
+        "key": "ctrl+i",
+        "command": "r.runCommandWithEditorPath",
+        "when": "editorTextFocus",
+        "args": "rmarkdown::render('$$', output_format = rmarkdown::html_document(), output_dir = \".\", clean = TRUE)"
```

Comment:
We may better use both double quotes or both single quotes in one command?

Reply:
I think it looks nicer than escaped double quotes if that's what you mean? It's how I would put it in my config file even if I don't normally use single quotes that often in R.

Reply:
But I suppose if you show some escaping it might make it clear this is necessary... okay from a instructable perspective it makes sense.

Reply:
Yes, single quotes look better here and do you mind making `output_dir = \".\"` use single quotes too to look more consistent?

Reply:
Also I just realised what you meant I think... I had a mix of singles AND escaped doubles.

Reply:
Yes, user needs to know `\"` is required to use double quotes here.